### PR TITLE
Update paths, and remove deprecated Facebook names

### PR DESCRIPTION
**Pull request by directionless**
Changes to update the various facebook names to the osquery ones.

I tried to get the paths as well. I might have missed a couple

Closes: #7159

### CODE REVIEW

**Comment by theopolis** [comment]
If you rename the launchctl related files then you'll also need a change in `osquery-packaging`, for example we could land this PR first https://github.com/osquery/osquery-packaging/pull/3/files then within your PR here, also update the pinned version of the `osquery/osquery-packaging` checkout within `.github/workflows/build_*.yml`

**Comment by theopolis** [comment]
Ok this should work if you:
1. Rebase onto master
2. Add to the PR, updating both of the locations below to be the new `osquery/osquery-packaging` HEAD: 372cc337d6dfe769cfd18bf24c13093225166ae0

https://github.com/osquery/osquery/blob/master/.github/workflows/build_aarch64.yml#L32
https://github.com/osquery/osquery/blob/master/.github/workflows/build_x86.yml#L36

**Comment by theopolis** [comment]
Looks like the three executable files need a `DESTINATION` of `/opt/osquery/bin`, here: https://github.com/osquery/osquery/pull/7210/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR469, and also with `osquery` and `osqueryctl`

**Review by theopolis — DISMISSED** [review]
(see comment above)

**Comment by theopolis** [comment]
Hey @sharvilshah and @directionless, where do the `Info.plist` and `PkgInfo` files here: https://github.com/osquery/osquery-packaging/commit/372cc337d6dfe769cfd18bf24c13093225166ae0#diff-c085fbac7afb94d6fd2c95f9df83f8c1d8a785122cafca95e0aa06785b9256e1R32 come from?

**Comment by sharvilshah** [comment]
Hey @theopolis they are from #7184

**Comment by theopolis** [comment]
Awesome, what do we need to do to merge that PR?

File: tools/windows_resources.rc.in [hunk 1]
```diff
@@ -7,7 +7,7 @@
 #define VER_PRODUCTVERSION          @osquery_version_major@,@osquery_version_minor@,@osquery_version_patch@,0
 #define VER_PRODUCTVERSION_STR      "@osquery_version@\0"
 
-#define VER_COMPANYNAME_STR         "Facebook"
+#define VER_COMPANYNAME_STR         "Osquery"
```
Comment (directionless) [inline tools/windows_resources.rc.in:10]:
```suggestion
#define VER_COMPANYNAME_STR         "Osquery Foundation"
```

**Comment by directionless** [comment]
Passes CI. I can see there are some issues in the resultant packages, but I'm inclined to merge this and let someone pick up #7236 and #7237 separately.

**Comment by alessandrogario** [comment]
I think the only missing part is to update osquery-packaging, then include the new submodule commit here so that we don't break the packages generated by the CI from master; currently, the .pkg archive does not install correctly

**Comment by directionless** [comment]
> I think the only missing part is to update osquery-packaging, then include the new submodule commit here so that we don't break the packages generated by the CI from master; currently, the .pkg archive does not install correctly

Huh? I thought we did that in #7184

**Comment by alessandrogario** [comment]
> > I think the only missing part is to update osquery-packaging, then include the new submodule commit here so that we don't break the packages generated by the CI from master; currently, the .pkg archive does not install correctly
> 
> Huh? I thought we did that in #7184

I am not sure if I am doing something wrong, but I have attempted to download and install the .pkg file generated by the CI, and it installed empty folders:

```
$ find /opt/osquery
/opt/osquery
/opt/osquery/lib
```

**Review by theopolis — APPROVED** [review]

**Pull request merged**